AutomatticTracks/build.gradle

```
@@ -38,16 +38,26 @@ android {
     }
 }
 
-
-publish {
-    userOrg = 'automattic'
-    groupId = 'com.automattic'
-    uploadName = 'tracks'
-    artifactId = 'tracks'
-    desc = 'Android client for Nosara tracks (event tracking and analytics)'
-    publishVersion = android.defaultConfig.versionName
-    licences = ['MIT']
-    website = 'https://github.com/Automattic/Automattic-Tracks-Android'
-    dryRun = 'false'
-    autoPublish = 'true'
+afterEvaluate {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                artifactId 'Automattic-Tracks-Android'
+                artifact("$buildDir/outputs/aar/AutomatticTracks-debug.aar")
```

I believe we can use `from components.release` instead.
```suggestion
                from components.release
```